xamarin/realm-tutorial-dotnet/TaskPage.xaml

```diff
@@ -0,0 +1,66 @@
+             xmlns:local="clr-namespace:RealmTemplateApp">
+    <NavigationPage.TitleView>
+        <StackLayout Orientation="Horizontal">
+            <Button x:Name="Logout" TextColor="#000" CornerRadius="5" Text="Log out" HorizontalOptions="Start" WidthRequest="100" 
+                    Clicked="Logout_Clicked"/>
+            <Label Text="Tasks" FontSize="18" TextColor="#000" FontAttributes="Bold" HorizontalOptions="CenterAndExpand" VerticalOptions="Center"/>
+            <Label Text="" HorizontalOptions="End" WidthRequest="100"/>
+        </StackLayout>
+    </NavigationPage.TitleView>
+    <ContentPage.Content>
+        <AbsoluteLayout HorizontalOptions="CenterAndExpand" VerticalOptions="CenterAndExpand">
+            <StackLayout x:Name="WaitingLayout"
+            AbsoluteLayout.LayoutBounds="0,0,1,1"
+            AbsoluteLayout.LayoutFlags="All"
+            BackgroundColor="Gray" Opacity="0.5">
+                <ActivityIndicator
+                x:Name="Waiting"
+                IsRunning="True"
+                HorizontalOptions="CenterAndExpand"
+                VerticalOptions="CenterAndExpand"
+                Color="Black"/>
+            </StackLayout>
+            <Frame BorderColor="Black" HorizontalOptions="StartAndExpand" Margin="-20,0,0,0">
+                <ScrollView>
+                    <ListView x:Name="listTasks">
+                        <ListView.ItemTemplate>
+                            <DataTemplate>
+                                <ViewCell>
+                                    <Grid Padding="10,0,15,0" BackgroundColor="#ceF">
+                                        <Grid.ColumnDefinitions>
+                                            <ColumnDefinition Width="4*"/>
+                                            <ColumnDefinition Width="1*"/>
+                                            <ColumnDefinition Width="1*"/>
+                                        </Grid.ColumnDefinitions>
+                                        <Label Text="{Binding Summary}" TextColor="Black" Grid.Column="0" VerticalOptions="Center"></Label>
+                                        <Switch x:Name="chkCompleted" IsToggled="{Binding IsComplete}" Grid.Column="1" VerticalOptions="Center"
+                                                Toggled="chkCompleted_Toggled" AutomationId="{Binding Id}"/>
+                                        <StackLayout Grid.Column="2" VerticalOptions="Center">
+                                            <Image WidthRequest="30" x:Name="btnDelete" Source="trash.png"
+                                                      VerticalOptions="CenterAndExpand" Aspect="AspectFit" AutomationId="{Binding Id}" >
+                                                <Image.GestureRecognizers>
+                                                    <TapGestureRecognizer Tapped="Delete_Clicked"/>
```

Comment:
(This is to make it easier to delete a task)
Change this to 
```
<TapGestureRecognizer Tapped="Delete_Clicked" CommandParameter="{Binding .}"/>
```
This is practically saying that you also need to pass the command parameter to the method. In this case the command parameter is ".", that is the same as the binding context for the list element, that in this case is the Task. 
Maybe it's not the easiest to wrap one's head around, but it really simplifies things :)